osu.Game/Screens/Edit/Components/Timelines/Summary/SummaryTimeline.cs

```diff
@@ -41,7 +43,7 @@ private void load(OsuColour colours)
                 {
                     Name = "centre line",
                     RelativeSizeAxes = Axes.Both,
-                    Colour = colours.Gray5,
+                    Colour = colourProvider.Background2,
```

Comment:
To confirm - is this an intentional deviation from [the version of design that I'm seeing](https://www.figma.com/file/ytnnne2TH8Z956Jxhiypqq/Beatmap-Editor-2?node-id=0%3A1), or is there a newer version that isn't available to me? Seems like design is using pure white/`Content1` there.

Reply:
The white felt a bit too bright, so I was experimenting with this. Will probably revert to white to match for now.

Reply:
I'm honestly okay (and can agree) with keeping as is, mostly asked out of curiosity.